packages/suite/src/actions/suite/__tests__/protocolActions.test.ts

```diff
@@ -12,7 +12,8 @@ export const getInitialState = (state?: ProtocolState) => ({
         ...protocolReducer(undefined, { type: 'foo' } as any),
         ...state,
     },
-    suite: {
+    suite: {},
```

Comment:
can be `suite` deleted here completely?

Reply:
Good point. Not needed anymore. https://github.com/trezor/trezor-suite/pull/9202/commits/a2ef83f59dfc5af75970129c02200383412955cb